mock/tenant_service.go

```diff
@@ -0,0 +1,44 @@
+package mock
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb/v2"
+	platform "github.com/influxdata/influxdb/v2"
+)
+
+// TenantService is a mock implementation of an authorization.TenantService
+type TenantService struct {
```

Comment:
the name here is confusing. I know it relates to your http specific tenant service implementation but there is also a influxdb.TenantService. I think this mock could be scoped to your http service.